lib/logger.ts

```diff
@@ -0,0 +1,47 @@
+import chalk from "chalk";
+
+class Log {
```

Comment:
We could consider making npm from this since we are using it in multiple repos

Reply:
Good thought, but we are actually only using this log class in this repo. The logging for the bot is specific to the backend server.